Resources/Audio/Animals/attributions.yml

```diff
@@ -117,24 +117,45 @@
   license: "CC-BY-4.0"
   copyright: "Audio is recorded/created by Pfranzen 'FreeSound.org'. The original audio was trimmed and renamed"
   source: "https://freesound.org/people/pfranzen/sounds/322744/"
-  
+
 - files: ["dog_bark1.ogg"]
   license: "CC0-1.0"
   copyright: "Audio is recorded/created by KFerentchak 'FreeSound.org'. The original audio was trimmed and renamed"
-  source: "https://freesound.org/people/KFerentchak/sounds/235912/"  
-  
+  source: "https://freesound.org/people/KFerentchak/sounds/235912/"
+
 - files: ["dog_bark2.ogg"]
   license: "CC0-1.0"
   copyright: "Audio is recorded/created by KFerentchak 'FreeSound.org'. The original audio was trimmed and renamed"
-  source: "https://freesound.org/people/KFerentchak/sounds/235912/"  
-  
+  source: "https://freesound.org/people/KFerentchak/sounds/235912/"
+
 - files: ["dog_bark3.ogg"]
   license: "CC0-1.0"
   copyright: "Audio is recorded/created by KFerentchak 'FreeSound.org'. The original audio was trimmed and renamed"
   source: "https://freesound.org/people/KFerentchak/sounds/235912/"
-  
+
 - files: ["nymph_chirp.ogg"]
   license: "CC-BY-SA-3.0"
   copyright: "Taken from ParadiseSS13"
   source: "https://github.com/ParadiseSS13/Paradise/commit/a34f1054cef5a44a67fdac3b67b811137c6071dd"
-  
\ No newline at end of file
+
+- files:
+    - fox1.ogg
+    - fox2.ogg
+    - fox3.ogg
+    - fox4.ogg
+    - fox5.ogg
+    - fox6.ogg
+    - fox7.ogg
+    - fox8.ogg
+    - fox9.ogg
+    - fox10.ogg
+    - fox11.ogg
+    - fox12.ogg
+    - fox13.ogg
+    - fox14.ogg
+    - fox15.ogg
+    - fox16.ogg
+    - fox_distort.ogg
+  copyright: "Created by fujiwaranao"
+  license: "CC BY-NC-SA 4.0"
```

Comment:
Replace the spaces in the license with dashes as per https://spdx.org/licenses/.

Should be `  license: "CC-BY-NC-SA-4.0"`

Reply:
oh thank you oopsie